lib/core/plugin/manager.js

```diff
@@ -401,7 +401,11 @@ class PluginsManager {
    * @param {string|function} fn - function to attach
    */
   registerPipe (plugin, warnDelay, event, fn) {
-    const _fn = typeof fn === 'function' ? fn : plugin.object[fn];
+    const _fn = typeof fn === 'function'
+      ? fn
+      // function names need to be bound to their object of origin to prevent
+      // invalid context execution
+      : plugin.object[fn].bind(plugin.object);
```

Comment:
Comment inside a ternary operator may be a little too unorthodox?
```suggestion
      // function names need to be bound to their object of origin to prevent
      // invalid context execution
    const _fn = typeof fn === 'function' ? fn : plugin.object[fn].bind(plugin.object);
```

Reply:
Probably.

Reply:
Fixed. :+1: